### PR TITLE
Required for Date calculation

### PR DESCRIPTION
**Pull request by truongwp**
This PR includes:
- Ignore `formidableforms1.css` file.
- Improve `dom.tag()` to accept a string as one of children.
- Add JS action when show inline modal (in the backend).
- Add `inside_class` to the inline modal args (to fix the ugly styling of nested modals inside the `<div class="inside">` tag).
- Bump version for testing.

### CODE REVIEW

**Comment by truongwp** [comment]
@Crabcyborg These are required codes for Date calculation. The PR in Pro hasn't been ready yet. But I think this is ready for review. Can you take a look when you have time?

**Comment by Crabcyborg** [comment]
@truongwp These changes all look fine to me!

File: classes/helpers/FrmAppHelper.php [hunk 1]
```diff
@@ -1024,6 +1024,7 @@ public static function kses_icon( $icon ) {
 	public static function add_allowed_icon_tags( $allowed_html ) {
 		$allowed_html['svg']['data-open'] = true;
 		$allowed_html['svg']['title']     = true;
+		$allowed_html['svg']['data-fid']  = true;
```
Comment (truongwp) [inline classes/helpers/FrmAppHelper.php:1027]: @Crabcyborg Do you think we should put a filter here then add `data-fid` in the dates plugin?

```
$allowed_svg_attrs = apply_filters( 'frm_allowed_svg_icon_attrs', array( 'data-open', 'title' ) );
```

Comment (truongwp) [reply classes/helpers/FrmAppHelper.php]: Using `frm_striphtml_allowed_tags` works. I updated the code. Thank you.

**Review by Crabcyborg — APPROVED** [review]
I'll merge this as well so we're ready for a datepicker release.

It looks good to me!

🚀

**Pull request merged**